tests/post_training/model_scope.py

```diff
@@ -8,86 +8,47 @@
-    model_scope.append({"name": "xception", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "efficientnet_b0", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append(
-        {
-            "name": "efficientnet_b0",
-            "report_model_name": "efficientnet_b0_BC",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "fast_bias_correction": False},
-        }
-    )
-    model_scope.append({"name": "darknet53", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    # ResNets
-    model_scope.append({"name": "resnest14d", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "inception_resnet_v2", "quantization_params": {}})
-    model_scope.append({"name": "wide_resnet50_2", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "regnetx_002", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    # MobileNets
-    model_scope.append({"name": "mobilenetv3_small_050", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    # Transformers
-    model_scope.append(
-        {
-            "name": "levit_128",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    model_scope.append(
-        {
-            "name": "deit3_small_patch16_224",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    model_scope.append(
-        {
-            "name": "swin_base_patch4_window7_224",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    # convit_tiny supressed due to bug - 104173
-    # model_scope.append({"name": "convit_tiny", "quantization_params": {"preset":QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER}})
-    model_scope.append(
-        {
-            "name": "visformer_small",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    model_scope.append(
-        {
-            "name": "crossvit_9_240",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    # Others
-    model_scope.append({"name": "hrnet_w18", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "efficientnet_lite0", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "dpn68", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "dla34", "quantization_params": {"preset": QuantizationPreset.MIXED}})
+    for model_name in list(model_scope.keys()):
```

Comment:
```suggestion
    for model_name in model_scope:
```

Reply:
It's no works if size of dict change in the loop. 
`model_scope.pop(model_name)`

Reply:
Then, it's a bad design of the method that changes the dictionary while walking on it.

Reply:
Why?